pkg/examples/containerized-workload/sample_application_config.yaml

```diff
@@ -0,0 +1,66 @@
+---
```

Comment:
Please split definition yamls from app_config.yaml, they do not belong to same world.